test/test_helper.rb

```diff
@@ -245,6 +245,26 @@ def statement_address(options = {})
       }.update(options)
     end
 
+    def stored_credential(*args, **options)
```

Comment:
Added this helper to make generating the stored credential hash less verbose. Since this is useful for any gateway we write stored credential tests for using the standardized hash format, I added it to the test helper directly.

Reply:
Nice, this also serves as good documentation of the format in code, since it is only otherwise present in the PR/commit message.